release/s3_sync.sh

```diff
@@ -44,6 +44,7 @@ cd ${PREFIX_DIR}
 for CRD
 in *yaml
 do
+  aws s3 ls s3://${ARTIFACT_BUCKET}/${PREFIX_DIR}/${CRD} ||
```

Comment:
The way this is spaced out this looks like 2 different commands. Could we make this one line or add a `\` or something to make it more clear its one command?

Reply:
Done